tests/Http/ParamsTest.php

```diff
@@ -90,6 +90,14 @@ public function testExtractFromSeparator(): void
 		$this->assertSame($expected, $params);
 	}
 
+	public function testMerge(): void
+	{
+		$params = new Params(['foo' => 'bar']);
+		$params->merge(['baz' => 'qux']);
+		$this->assertSame('bar', $params->foo);
+		$this->assertSame('qux', $params->baz);
```

Comment:
`Suggestion`: To ensure that existing keys are overridden in the intended order:

```suggestion
		$params = new Params(['foo' => 'bar', 'bar' => 'baz']);
		$params->merge(['bar' => 'foo', 'baz' => 'qux']);
		$this->assertSame('bar', $params->foo);
		$this->assertSame('foo', $params->bar);
		$this->assertSame('qux', $params->baz);
```

Reply:
Included it manually when rebasing the PR.